code/modules/spells/spell_types/pointed/eoran_charm.dm

```diff
@@ -41,7 +41,8 @@
 	var/list/charms_target = list("<b style='color:pink'>Your eyes cannot move away from [owner].</b>", "<b style='color:pink'>You are enchanted by the beauty of the follower of Eora.</b>")
 	cast_on.visible_message(span_warning("[pick(charms_public)]"), span_warning("[pick(charms_target)]"))
 	cast_on.apply_status_effect(/datum/status_effect/eorapacify)
-	cast_on.Immobilize(85)
+	cast_on.apply_status_effect(/datum/status_effect/debuff/mesmerised)
```

Comment:
This is 3x the length of the pacify effect and 8x the length of the stun, you can specfiy a time override to status effects as the second variable